app/src/main/java/com/example/weatherapp/view/ChooseCityFragment.kt

```diff
@@ -0,0 +1,138 @@
+package com.example.weatherapp.view
+
+import android.app.AlertDialog
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.EditText
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.example.weatherapp.CityAdapter
+import com.example.weatherapp.CityInterface
+import com.example.weatherapp.R
+import com.example.weatherapp.viewModel.WeatherViewModel
+import com.example.weatherapp.common.contract
+import com.example.weatherapp.databinding.FragmentChooseCityBinding
+
+class ChooseCityFragment : Fragment(R.layout.fragment_choose_city) {
+    private lateinit var binding: FragmentChooseCityBinding
```

Comment:
clear link to your binding in onDestroy 
according with https://developer.android.com/topic/libraries/view-binding